app/src/main/res/values-tr/strings.xml

```diff
@@ -20,7 +20,7 @@
     <string name="settingsDefaultBrowserEnabled">Varsayılan tarayıcı olarak ayarla</string>
     <string name="defaultBrowserMaybeLater">Belki Sonra</string>
     <string name="defaultBrowserLetsDoIt">Varsayılan tarayıcı olarak ayarla</string>
-    <string name="defaultBrowserDescriptionNoDefault">Her zaman gönül rahatlığıyla açık bağlantılar.</string>
+    <string name="defaultBrowserDescriptionNoDefault">Her zaman gönül rahatlığıyla bağlantıları açın.</string>
```

Comment:
Sorry, I'm confused now. I thought you suggested changing `bağlantıları` for another word that means Link, why have you decided to use the previous one? is more polite?

Reply:
Oh sorry, I've forgotten to mention about that. It's not polite but officially accepted translated word in the Turkish dictionary, even `Link` is the mostly used, I think we should stick with the official dictionary, I guess.

Reply:
"açın" was already polite. "gir" was informal, so, I changed to "girin" to make it formal. Also sounds more polite, and even there's more polite way is "giriniz" but that would be too much :)

Reply:
Ok thanks, now it's clear. Yes, as commented above, better if we only fix typos and grammatical errors, so let's keep it as it is right now.
The word for link seems correct based on your last comment even if we could use a better one.